### PR TITLE
Badge and Button Component Added

### PR DESCRIPTION
**Pull request by keshavkumar-in**

### CODE REVIEW

File: docs/components/badge/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      style="
+        width: 100%;
+        max-width: 100%;
+        height: 445px;
```
Comment (Developers-codz) [inline docs/components/badge/index.html]: do not use px if it has large value

File: docs/components/button/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  </div>
+  <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=915.5&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=34px&ph=52px&ln=true&fl=1&fm=Hack&fs=15.5px&lh=163%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522button%2520button-primary-outline%2522%253EPrimary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-success-outline%2522%253ESuccess%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-info-outline%2522%253EInfo%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-warning-outline%2520button-round%2522%253EWarning%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-danger-outline%2522%253EDanger%253C%252Fbutton%253E"
+    style="
+      width: 100%;
+      max-width: 100%;
+      height: 268px;
+      border: 0;
+      transform: scale(1);
+      overflow: hidden;
```
Comment (Developers-codz) [inline docs/components/button/index.html]: I think one should not use inline css so make a common  class for every code snippets

File: lib/package/components/button.css [hunk 3]
```diff
@@ -2,36 +2,131 @@
...
+}
+
+.button-warning-outline {
+  color: var(--bg-warning);
+  background-color: transparent;
+  border: 2px solid var(--bg-warning);
+}
+
+.button-warning-outline:hover {
+  color: #fff;
```
Comment (Developers-codz) [inline lib/package/components/button.css]: don't directly use color code here. Make variable

**Review by Developers-codz — APPROVED** [review]

**Review by MayankKumar10 — APPROVED** [review]
Look into what kajal has suggested, Everything looks good you can merge it.

File: lib/package/components/button.css (hunk 3 above)
Comment (keshavkumar-in) [reply lib/package/components/button.css]: Thank You Kajal, made changes as you suggested.

**Comment by keshavkumar-in** [comment]
> Look into what kajal has suggested, Everything looks good you can merge it.

Yeah, made the changes. Thank Your for reviewing bro.

**Pull request merged**